### PR TITLE
taskell 1.2.6 (new formula)

### PR DESCRIPTION
**Pull request by katjad**
- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/master/CONTRIBUTING.md)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install <formula>`)?

-----

### CODE REVIEW

File: Formula/taskell.rb [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+  def install
+    cabal_sandbox do
+      cabal_install "hpack"
+      system "./.cabal-sandbox/bin/hpack"
+      install_cabal_package
+    end
+  end
+
+  test do
+    system "#{bin}/taskell", "-v"
```
Comment (fxcoudert) [inline Formula/taskell.rb]: We need a test that exercises the some of the functionality of the app.

**Review by fxcoudert — CHANGES_REQUESTED** [review]

File: Formula/taskell.rb (hunk 1 above)
Comment (smallhadroncollider) [reply Formula/taskell.rb]: I'll need to add some functionality to taskell so that it outputs something testable to stdout.

**Comment by ilovezfs** [comment]
Thanks so much for your first contribution! Without people like you submitting PRs we couldn't run this project. You rock, @katjad!

**Comment by smallhadroncollider** [comment]
Now this is in `homebrew-core` is there some way I can update my unofficial [`smallhadroncollider/taskell/taskell`](https://github.com/smallhadroncollider/homebrew-taskell/blob/master/Formula/taskell.rb) repo to redirect to the `homebrew-core` version?

Should I just update it to `odie` with a message about using the official version?

**Comment by ilovezfs** [comment]
You can use `tap_migrations.json` as in https://github.com/dinkypumpkin/homebrew-get_iplayer

**Pull request merged**